packages/framework-provider-local/test/services/read-model-registry.test.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { ReadModelEnvelope } from '@boostercloud/framework-types'
+import { expect } from '../expect'
+import * as faker from 'faker'
+import { stub, restore } from 'sinon'
+import { ReadModelRegistry } from '../../src/services'
+import { createMockReadModelEnvelope } from '../helpers/read-model-helper'
+import { random } from 'faker'
+
+describe('the read model registry', () => {
+  let initialReadModelsCount: number
+  let mockReadModel: ReadModelEnvelope
+
+  let readModelRegistry: ReadModelRegistry
+
+  beforeEach(async () => {
+    initialReadModelsCount = random.number({ min: 2, max: 10 })
+    readModelRegistry = new ReadModelRegistry()
```

Comment:
If you're using the actual database, this could be considered an "integration test" and the real I/O will likely make it slower. Would it make sense to mock the DB or use the in-memory version of nedb?

Reply:
I'll approach this on a new PR